options/locale/locale_en-US.ini

```diff
@@ -3338,6 +3338,10 @@ alpine.repository = Repository Info
 alpine.repository.branches = Branches
 alpine.repository.repositories = Repositories
 alpine.repository.architectures = Architectures
+arch.pacmanconf = Add server with related distribution and architecture to <code>/etc/pacman.conf</code>:
+arch.pacmansync = Sync package with pacman:
+arch.documentation = For more information on the arch mirrors, see <a target="_blank" rel="noopener noreferrer" href="%s">the documentation</a>.
```

Comment:
Ideally we should not have HTML in the translation because later changes to the HTML are hard. One way of avoiding is to put the whole start and end tags token instead so this string becomes:

```ini
arch.documentation = For more information on the arch mirrors, see %sthe documentation%s.
```

And  the usage is:

```handlebars
{{.locale.Tr "packages.arch.documentation" (printf `<a target="_blank" rel="noopener noreferrer" href="https://docs.gitea.io/en-us/usage/packages/arch/">`) (printf </a>) | Safe}}